### ISSUE TITLE
`khr::RayTracing::cmd_trace_rays`

### ISSUE THREAD

**Issue by zakarumych** [MaikKlein/ash#304]
[`khr::RayTracing::cmd_trace_rays`](https://docs.rs/ash/0.31.0/ash/extensions/khr/struct.RayTracing.html#method.cmd_trace_rays) should accept references to [`vk::StridedBufferRegionKHR`](https://docs.rs/ash/0.31.0/ash/vk/struct.StridedBufferRegionKHR.html), and not to slices.

See https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCmdTraceRaysKHR.html
Particularly:
```
• pRaygenShaderBindingTable must be a valid pointer to a valid VkStridedBufferRegionKHR structure
• pMissShaderBindingTable must be a valid pointer to a valid VkStridedBufferRegionKHR structure
• pHitShaderBindingTable must be a valid pointer to a valid VkStridedBufferRegionKHR structure
• pCallableShaderBindingTable must be a valid pointer to a valid VkStridedBufferRegionKHR structure
```

**Comment by zakarumych** [comment]
It is not a block as one can use [`std::slice::from_ref`](https://doc.rust-lang.org/std/slice/fn.from_ref.html) to convert reference to a slice.

**Comment by nickkuk** [comment]
This was fixed in 84624fddd809ef13af4061e154ed130d950326bf.

**Comment by MarijnS95** [comment]
As mentioned this was fixed in #350 :)
